titus-ext/aws/src/main/java/com/netflix/titus/ext/aws/apigateway/ScalingPayload.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.titus.ext.aws.apigateway;
+
+public class ScalingPayload {
```

Comment:
Mentioned above but would be great to use the builder pattern here to be consistent.